### DISCUSSION TITLE
Get old Olive back

### DISCUSSION THREAD

**Discussion by Oliveisnotcool** [olive-editor/olive#2181] [Q&A]
**Commit Hash** 

**Platform** 

**Summary**

**Additional Information / Output**

**Comment by Oliveisnotcool** [comment]
I just want beta olive i deleted olive got it back and this new olive is not for me pls somone give me a link to the beta old olive the og one i just want it back so badly i cant make videos

**Reply by Simran-B** [reply]
Please create issues for bug report only and provide sufficient information so that people can actually help you.
I can only guess that you want to download Olive 0.1, which you can on the website:

<img width="614" alt="image" src="https://user-images.githubusercontent.com/7819991/221284140-a2b848e5-2af7-4ac6-8ab9-792fcdafa933.png">

https://olivevideoeditor.org/download.php
